### PR TITLE
DEV-1409: Add sunset notice for dwolla.js to changelog and related docs

### PR DESCRIPTION
**Pull request by ShreyaThapa**

### CODE REVIEW

File: pages/api-reference/funding-sources/create-a-funding-sources-token-for-dwolla-js.mdx [hunk 1]
```diff
@@ -13,6 +13,13 @@ Dwolla.js is a client-side library that allows you to `add` a funding source for
 
 Refer to our [Dwolla.js developer guide](https://developers.dwolla.com/guides/dwolla-js/add-a-bank-account) to learn more on how to utilize this single-use funding-sources-token to `add` funding source for a Customer.
 
+<AlertBar variation="announcement">
+  Deprecation Announcement: Dwolla will be discontinuing support for dwolla.js on August 31, 2023. This library is no longer being actively developed, and we recommend using the <a href="/concepts/drop-in-components">Drop-in Components library</a> instead. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```
Comment (spencerhunter) [inline pages/api-reference/funding-sources/create-a-funding-sources-token-for-dwolla-js.mdx]: @ShreyaThapa slight tweak on the language to use Sunset vs. Deprecation. 

Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.

File: pages/changelog.mdx [hunk 2]
```diff
@@ -12,6 +12,15 @@ Keep up to date on new product features and enhancements, API docs, tools change
 
 ## Upcoming
 
+### 2023-08-31
+
+**DEPRECATED**
```
Comment (spencerhunter) [inline pages/changelog.mdx]: **SUNSET**

- Dwolla will be retiring the `dwolla.js` library, which supports adding an unverified bank funding source. Check out the official [announcement](https://discuss.dwolla.com/t/may-2023-updates-retiring-dwolla-s-client-side-javascript-library-dwolla-js/8969).
- The legacy dwolla.js will be replaced with functionality that has been added to our [Drop-in Components library](/concepts/drop-in-components#latest-v220). This library will serve as our primary web UI components library and offer an enhanced developer experience. Please refer to our [migration guide](https://discuss.dwolla.com/t/migration-guide-dwolla-js-to-dwolla-web-js/8968) for more details, including alternative solutions.
-  The following API endpoints will be removed as part of the sunset: `/customers/{id}/iav-token` and `/customers/{id}/funding-sources-token`

**Review by spencerhunter — CHANGES_REQUESTED** [review]

File: pages/api-reference/funding-sources/create-a-funding-sources-token-for-dwolla-js.mdx (hunk 1 above)
Comment (ShreyaThapa) [inline pages/api-reference/funding-sources/create-a-funding-sources-token-for-dwolla-js.mdx]:
```suggestion
  Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```

File: pages/changelog.mdx [hunk 3]
```diff
@@ -12,6 +12,15 @@ Keep up to date on new product features and enhancements, API docs, tools change
...
+
+- Dwolla will be discontinuing support for `dwolla.js` library, which supports adding an unverified bank funding source. Check out the official [announcement](https://discuss.dwolla.com/t/may-2023-updates-retiring-dwolla-s-client-side-javascript-library-dwolla-js/8969).
+- We are replacing this legacy library with functionality that has been added to our [Drop-in Components library](/concepts/drop-in-components#latest-v220). This library will serve as our primary web UI components library and offer an enhanced developer experience. Please refer to our [migration guide](https://discuss.dwolla.com/t/migration-guide-dwolla-js-to-dwolla-web-js/8968) for more details, including alternative solutions.
```
Comment (ShreyaThapa) [inline pages/changelog.mdx]:
```suggestion
**SUNSET**

- Dwolla will be retiring the `dwolla.js` library, which supports adding an unverified bank funding source. Check out the official [announcement](https://discuss.dwolla.com/t/may-2023-updates-retiring-dwolla-s-client-side-javascript-library-dwolla-js/8969).
- The legacy dwolla.js will be replaced with functionality that has been added to our [Drop-in Components library](/concepts/drop-in-components#latest-v220). This library will serve as our primary web UI components library and offer an enhanced developer experience. Please refer to our [migration guide](https://discuss.dwolla.com/t/migration-guide-dwolla-js-to-dwolla-web-js/8968) for more details, including alternative solutions.
- The following API endpoints will be removed as part of the sunset: `/customers/{id}/iav-token` and `/customers/{id}/funding-sources-token`.
```

File: pages/concepts/bank-funding-source.mdx [hunk 4]
```diff
@@ -70,6 +70,13 @@ Adding a bank with the Dwolla API is easy. Take a look at our documentation to l
 
 ### Dwolla.js
 
+<AlertBar variation="announcement">
+  Deprecation Announcement: Dwolla will be discontinuing support for dwolla.js on August 31, 2023. This library is no longer being actively developed, and we recommend using the <a href="/concepts/drop-in-components">Drop-in Components library</a> instead. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```
Comment (ShreyaThapa) [inline pages/concepts/bank-funding-source.mdx]:
```suggestion  
  Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```

File: pages/guides/dwolla-js/add-a-bank-account/configure-and-call-function.mdx [hunk 5]
```diff
@@ -8,6 +8,13 @@ meta:
   description: Set up dwolla.js configuration and function to create a new funding source.
 ---
 
+<AlertBar variation="announcement">
+  Deprecation Announcement: Dwolla will be discontinuing support for dwolla.js on August 31, 2023. This library is no longer being actively developed, and we recommend using the <a href="/concepts/drop-in-components">Drop-in Components library</a> instead. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```
Comment (ShreyaThapa) [inline pages/guides/dwolla-js/add-a-bank-account/configure-and-call-function.mdx]:
```suggestion
  Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```

File: pages/guides/dwolla-js/add-a-bank-account/create-form.mdx [hunk 6]
```diff
@@ -8,6 +8,13 @@ meta:
   description: Create the form that will be used as inputs for the account and routing number.
 ---
 
+<AlertBar variation="announcement">
+  Deprecation Announcement: Dwolla will be discontinuing support for dwolla.js on August 31, 2023. This library is no longer being actively developed, and we recommend using the <a href="/concepts/drop-in-components">Drop-in Components library</a> instead. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```
Comment (ShreyaThapa) [inline pages/guides/dwolla-js/add-a-bank-account/create-form.mdx]:
```suggestion
  Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```

File: pages/guides/dwolla-js/add-a-bank-account/create-token.mdx [hunk 7]
```diff
@@ -8,6 +8,13 @@ meta:
   description: Create a funding sources token for use in initiating Dwolla.js, a powerful and secure bank account verification tool.
 ---
 
+<AlertBar variation="announcement">
+  Deprecation Announcement: Dwolla will be discontinuing support for dwolla.js on August 31, 2023. This library is no longer being actively developed, and we recommend using the <a href="/concepts/drop-in-components">Drop-in Components library</a> instead. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```
Comment (ShreyaThapa) [inline pages/guides/dwolla-js/add-a-bank-account/create-token.mdx]:
```suggestion
  Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```

File: pages/guides/dwolla-js/add-a-bank-account/handle-response.mdx [hunk 8]
```diff
@@ -8,6 +8,13 @@ meta:
   description: Handle a request's success and errors in dwolla.js when adding a bank.
 ---
 
+<AlertBar variation="announcement">
+  Deprecation Announcement: Dwolla will be discontinuing support for dwolla.js on August 31, 2023. This library is no longer being actively developed, and we recommend using the <a href="/concepts/drop-in-components">Drop-in Components library</a> instead. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```
Comment (ShreyaThapa) [inline pages/guides/dwolla-js/add-a-bank-account/handle-response.mdx]:
```suggestion
  Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```

File: pages/guides/dwolla-js/add-a-bank-account/index.mdx [hunk 9]
```diff
@@ -18,6 +18,13 @@ meta:
 
 Using dwolla.js, securely transmit sensitive data (bank account and routing number) to Dwolla without the data passing through your server. This resource guide will walk you through the process of generating a funding sources token, collecting the user's bank account information, and calling a JavaScript function to send this data to Dwolla. When you collect and submit the user's bank account information, dwolla.js has built-in validation that will trigger an error if any of the required fields are invalid.
 
+<AlertBar variation="announcement">
+  Deprecation Announcement: Dwolla will be discontinuing support for dwolla.js on August 31, 2023. This library is no longer being actively developed, and we recommend using the <a href="/concepts/drop-in-components">Drop-in Components library</a> instead. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```
Comment (ShreyaThapa) [inline pages/guides/dwolla-js/add-a-bank-account/index.mdx]:
```suggestion
  Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```

File: pages/guides/transfer-money-me-to-me/add-funding-sources.mdx [hunk 10]
```diff
@@ -24,6 +24,13 @@ There are multiple ways of adding a bank to a Customer with the Dwolla API. A si
 | [Dwolla + Secure Exchange solution](/concepts/secure-exchange) | Yes                           | Online banking credentials      |
 | Other Approved Third-party Provider                            | Yes                           | Variable                        |
 
+<AlertBar variation="announcement">
+  Deprecation Announcement: Dwolla will be discontinuing support for dwolla.js on August 31, 2023. This library is no longer being actively developed, and we recommend using the <a href="/concepts/drop-in-components">Drop-in Components library</a> instead. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```
Comment (ShreyaThapa) [inline pages/guides/transfer-money-me-to-me/add-funding-sources.mdx]:
```suggestion
  Sunset notice: Dwolla will be retiring dwolla.js on August 31, 2023. This library should no longer be used, and we recommend migrating to the latest client-side JavaScript library that includes <a href="/concepts/drop-in-components">Drop-in Components</a>. The Drop-in Components library offers an enhanced developer experience and includes functionality that was previously only available in dwolla.js.
```

File: pages/api-reference/funding-sources/create-a-funding-sources-token-for-dwolla-js.mdx (hunk 1 above)
Comment (ShreyaThapa) [reply pages/api-reference/funding-sources/create-a-funding-sources-token-for-dwolla-js.mdx]: [Applied suggestions from code review](https://github.com/Dwolla/dev-portal/pull/507/commits/4854c1cc3319348121c30188fb039396eda16222)

File: pages/changelog.mdx (hunk 2 above)
Comment (ShreyaThapa) [reply pages/changelog.mdx]: [Applied suggestions from code review](https://github.com/Dwolla/dev-portal/pull/507/commits/4854c1cc3319348121c30188fb039396eda16222)

**Review by spencerhunter — APPROVED** [review]

**Pull request merged**